### DISCUSSION TITLE
2023/02/ggplot2-cookbook/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#82] [General]
# 2023/02/ggplot2-cookbook/

https://xiangyun.rbind.io/2023/02/ggplot2-cookbook/

**Comment by tcgriffith** [comment]
标题 “配方”会不会比 食谱恰当一点？

**Reply by XiangyunHuang** [reply]
我大概是受 《[R Graphics cookbook](https://r-graphics.org/)》 的中文翻译影响。每一节都是一个操作（一道菜），比如插入公式，小节的里面就是介绍怎么插入公式。
